src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -213,6 +247,9 @@ class MockClientBackend : public ClientBackend {
     local_completed_req_count_++;
     stats_->num_active_infer_calls--;
 
+    stats_->recorded_inputs.insert(
```

Comment:
This only captures for normal Infer().  Why don't you put the code in stats->CaptureRequest instead? That way it will work for all the Infer functions.

Reply:
Moved 👍